### PR TITLE
sys-apps/tuned: Add 2.13.0 and use Python 3

### PR DESCRIPTION
**Pull request by ekohl**
Changes to tuned:
* Move from Python 2 to Python 3.
* Correct DOCDIR patching
* Optimize Python files
* Update XDG icon cache

This also includes dev-python/python-linux-procfs version 0.6.1 which adds Python 3 support and is a dependency. I'm not sure about the convention in this repository whether it's ok to combine both in a single PR.

### CODE REVIEW

File: sys-apps/tuned/tuned-2.13.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+inherit python-single-r1 systemd xdg-utils
+
+DESCRIPTION="Daemon for monitoring and adaptive tuning of system devices"
+HOMEPAGE="https://fedorahosted.org/tuned/"
+SRC_URI="https://github.com/redhat-performance/tuned/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
```
Comment (Zlogene) [inline sys-apps/tuned/tuned-2.13.0.ebuild]: keyword also for `~x86`

File: sys-apps/tuned/tuned-2.13.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+CDEPEND="
```
Comment (Zlogene) [inline sys-apps/tuned/tuned-2.13.0.ebuild]: rename to `COMMON_DEPEND`

File: sys-apps/tuned/tuned-2.13.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+DEPEND="
+	${CDEPEND}"
+
+RDEPEND="
+	${CDEPEND}
+	sys-apps/dbus
+	sys-apps/ethtool
+	sys-power/powertop
+	sys-process/procps
```
Comment (Zlogene) [inline sys-apps/tuned/tuned-2.13.0.ebuild]: you do not need this dependency

File: sys-apps/tuned/tuned-2.13.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (Zlogene) [inline sys-apps/tuned/tuned-2.13.0.ebuild]: why not 7?

File: sys-apps/tuned/tuned-2.13.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit python-single-r1 systemd xdg-utils
```
Comment (Zlogene) [inline sys-apps/tuned/tuned-2.13.0.ebuild]: `systemd` eclass is not used in the ebuild

File: sys-apps/tuned/tuned-2.13.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+		-e "/\$(DESTDIR)\/var\/lib\/tuned/d" \
+		-e "/\$(DESTDIR)\/var\/log\/tuned/d" \
+		Makefile || die
+}
+
+src_install() {
+	default
+
+	newinitd "${FILESDIR}/${PN}.initd" "${PN}"
+	python_fix_shebang "${ED}"
```
Comment (Zlogene) [inline sys-apps/tuned/tuned-2.13.0.ebuild]: `D` should be enough here, prefix is not used anywhere in the ebuild.

File: sys-apps/tuned/tuned-2.13.0.ebuild (hunk 1 above)
Comment (ekohl) [reply sys-apps/tuned/tuned-2.13.0.ebuild]: I've copied tuned-2.10.0 which already dropped it. ced5658b20ceb2b385067f4dc3c5546eb50929c3 states:
> 2) drop x86 keyword due to dev-python/python-linux-procfs

File: sys-apps/tuned/tuned-2.13.0.ebuild (hunk 2 above)
Comment (ekohl) [reply sys-apps/tuned/tuned-2.13.0.ebuild]: Also due to copying. Fixing.

File: sys-apps/tuned/tuned-2.13.0.ebuild (hunk 3 above)
Comment (ekohl) [reply sys-apps/tuned/tuned-2.13.0.ebuild]: Also due to copying. How do you determine this isn't needed? Reading the spec file for dependencies?

File: sys-apps/tuned/tuned-2.13.0.ebuild (hunk 4 above)
Comment (ekohl) [reply sys-apps/tuned/tuned-2.13.0.ebuild]: Because I copied the file and didn't think about it.

File: sys-apps/tuned/tuned-2.13.0.ebuild (hunk 3 above)
Comment (Zlogene) [reply sys-apps/tuned/tuned-2.13.0.ebuild]: it is a part of the system set

**Comment by ekohl** [comment]
I've pushed a separate commit that addresses the feedback for easy review. If this addresses the feedback I'll squash and force push with an updated message.

File: sys-apps/tuned/tuned-2.13.0.ebuild (hunk 2 above)
Comment (a17r) [reply sys-apps/tuned/tuned-2.13.0.ebuild]: however, `DEPEND` contains none else than `COMMON_DEPEND` so this var is just superfluous. Just make it `DEPEND` here already and inherit that in `RDEPEND`.

**Pull request closed without merging**